docs/main/scenario-tools/creating-and-updating.md

```diff
@@ -24,8 +24,9 @@ In the `scenario_list.csv` file you can create a row for each scenario you wish
  * **end_year**. The target year / year of interest of each scenario.
  * **description**. Scenario description. This is displayed in the model’s front-end.
  * **id**. Should be left empty if you want to create a new scenario. The ID will be filled in automatically when the scenario is created. If you want to update an existing scenario, you can specify its ID here.
- * **curve_file**. The name of a CSV file containing custom hourly profiles. For example interconnector price curves, solar production curves or industry heat demand curves. The CSV file should be placed in the `input/curves` folder.
+ * **keep_compatible**. Boolean value. Sets whether the scenario should be changed automatically if we make changes to the model, to keep it compatible.
```

Comment:
```suggestion
 * **keep_compatible**. True or False. Sets whether the scenario should be updated automatically if we make changes to the model; keeping your scenario compatible with future versions of the ETM ([read more](api/scenarios#forward-compatibility)).
```

I think users of these tools are not always aware of what a Boolean means, so it might be better to change it into true or false. Also, we can point them to the expert docs on compatibility with a 'read more', or do you think that is too advanced for them?

Reply:
Totally agree with your suggestions :) I think adding the 'read more' link is always a good thing! People can always decide for themselves whether they are interested in learning more, and whether it's too advanced or not.